### DISCUSSION TITLE
Remove some icons from Help menu

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#8326] [Editor]
Everything has an icon = nothing has an icon, especially when they look similar.

<table>
<tr><th>Current</th><th>A</th><th>B</th></tr>
<tr>
<td>

![ksnip_20231101-145700](https://github.com/godotengine/godot-proposals/assets/372476/4386b3ce-807a-4d39-9119-efefb5012850)
</td><td>

![ksnip_20231101-145701](https://github.com/godotengine/godot-proposals/assets/372476/dda2b6e0-d531-4a8d-9b47-1221e5b5ea87)
</td><td>

![ksnip_20231101-145703](https://github.com/godotengine/godot-proposals/assets/372476/ab7ce8a0-f294-4ef5-b0b4-154e694bbe14)
</td>
</tr>
</table>

* Currently, everything has an icon. It makes it easier to notice the relationship between items. But can't draw attention to important ones.
* It's easier to notice "Copy System Info" in A.
* I suggest changing into something like B.

**Comment by AThousandShips** [comment]
I'd suggest putting the external link icon on the right instead

**Reply by azur-wolve** [reply]
or let the links be underlined with blue color

**Reply by Calinou** [reply]
It's not possible for specific PopupMenu items to have underlined text, so this can't be done. Also, underlining links is something that's usually only done in prose text (RichTextLabel or LinkButton), not in menus.

I'd either remove the icons as proposed, or make the external link icon more subtle (e.g. make it look like this: **↗**)

**Edit:** Pull request opened: https://github.com/godotengine/godot/pull/85978
